src/main/java/fi/helsinki/cs/tmc/core/TmcCore.java

```diff
@@ -1,29 +1,11 @@
 package fi.helsinki.cs.tmc.core;
 
-import static com.google.common.base.Strings.isNullOrEmpty;
+import static fi.helsinki.cs.tmc.core.commands.factory.CommandFactory.*;
```

Comment:
Sorry, no .* imports: https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports
Event though it make the import list longer..